cities/conf.py

```diff
@@ -363,3 +365,4 @@ def create_plugins():
 # Users who want better postal codes can flip this on (developers of
 # django-cities itself probably will), but most probably won't want to
 VALIDATE_POSTAL_CODES = getattr(django_settings, 'CITIES_VALIDATE_POSTAL_CODES', False)
+DJANGO_VERSION = float('.'.join(map(str, django.VERSION[:2])))
```

Comment:
This doesn't relate to cities configuration, so it doesn't belong here in the first place.

Additionally, a better way to do this is to use `django.VERSION`, which returns a tuple and compare that to a version tuple, like so:`django.VERSION < (2, 0)`.